atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -258,9 +333,9 @@ private static SerializableTransactionManager create(
                 transactionManager,
                 kvs,
                 sweepRunner,
-                Suppliers.ofInstance(config.enableSweep()),
-                Suppliers.ofInstance(config.getSweepPauseMillis()),
-                Suppliers.ofInstance(getSweepBatchConfig(config)),
+                () -> MoreObjects.firstNonNull(runtimeConfig.get().enableSweep(), config.enableSweep()),
+                () -> MoreObjects.firstNonNull(runtimeConfig.get().getSweepPauseMillis(), config.getSweepPauseMillis()),
+                () -> getSweepBatchConfig(runtimeConfig.get(), config),
```

Comment:
This was addressed by adding a TODO on the AtlasDbConfig, to have add a default value on the new config when deleting the old config. The unit test doesn't seem valuable after we delete the old config. If people still think it would be good to add it, please let me know and I'll do so.